### PR TITLE
Cookie consent solution added to styles page 

### PR DESCRIPTION
**Pull request by joeedhorton**
### Two decisions have been made. 

- **to have the link re-directed the github issue itself rather than the specific GitHub repo with specific guidance.** This is because we shouldn't assume witch GitHub repo they need (wordpress, nodejs, salesforce etc.)
- **to have the link open in a new tab**. We don't want them to be taken away from the design system if they find a problem or didn't find what they were looking for.

### CODE REVIEW

**Comment by MarionDownie** [comment]
Hi Joe, see the guidance on this in the GOV.UK Design System - https://design-system.service.gov.uk/styles/typography/#links We should avoid opening links in a new tab for accessibility reasons.

**Comment by joeedhorton** [comment]
All done 👍 removed the `target="blank"`

File: app/views/styles/index.html [hunk 1]
```diff
@@ -100,24 +100,39 @@ <h2 class="govuk-heading-m">GOV.UK design styles</h2>
 
       <h2 class="govuk-heading-m">Independent branding</h2>
 
+      <div class="govuk-grid-row" style="margin: 24px 0px 32px 0px;">
+          <div class="govuk-grid-column-full" style="padding: 24px 32px; background-color: #e4e4e4; border-radius: 3px;">
+            <p style="font-weight: 700; font-size: 16px; margin-bottom: 4px;">Cookie consent solution</p>
+            <p class="govuk-body-s" style="font-size: 16px; margin: 0px; max-width: 40em;">
+              If you are a marketing or informational site looking to implement a cookie consent solution, please refer to the 
+                <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/216">
+                  documentation written by the cabinet office.
```
Comment (zuzak) [inline app/views/styles/index.html:109]:
```suggestion
                  documentation written by the Cabinet Office.
```

Comment (joeedhorton) [reply app/views/styles/index.html:109]: Good spot @zuzak 👍

**Comment by MarionDownie** [comment]
Had a look at this in the browser and looks fine, but I feel like it's too prominent - can the banner go at the bottom, below the main content for the styles? It is secondary to the main aim of the page, which is to find out which styling to use where.

**Review by karypun — CHANGES_REQUESTED** [review]
Could we amend "marketing and informational site" content as that is only one specific criteria making up the independent branding style. The cookie guidance applies to the other criteria listed too.  Could say something like, "For independently branded sites such as public-facing services not residing on GOV.UK, you must refer to the documentation written by the Cabinet Office." This should cover the criteria listed.

**Comment by MarionDownie** [comment]
> Could we amend "marketing and informational site" content as that is only one specific criteria making up the independent branding style. The cookie guidance applies to the other criteria listed too. Could say something like, "For independently branded sites such as public-facing services not residing on GOV.UK, you must refer to the documentation written by the Cabinet Office." This should cover the criteria listed.

This sounds ok, though can we just remove the word 'residing'? Ta

**Review by karypun — APPROVED** [review]

**Pull request merged**